src/main/java/hudson/plugins/performance/PerformancePublisher.java

```diff
@@ -147,7 +180,12 @@ public PerformancePublisher(int errorFailedThreshold,
                             boolean modeOfThreshold,
                             boolean compareBuildPrevious,
                             List<? extends PerformanceReportParser> parsers,
-                            boolean modeThroughput) {
+                            boolean modeThroughput,
+                            boolean modeEvaluation,
+                            List<? extends AbstractConstraint> constraints,
+                            boolean ignoreFailedBuilds,
+                            boolean ignoreUnstableBuilds,
+                            boolean persistConstraintLog) {
```

Comment:
This change breaks the binary compatibility. I would recommend to retain the original constructor and to mark it deprecated. <code>@DataBoundSetter</code> is an alternate option especially if you don't want to declare fields as final ones